### ISSUE TITLE
Stalfos cause the game to freeze completely 

### ISSUE THREAD

**Issue by LilaHax** [blawar/ooot#103]
## Current Behavior
When encountering stalfos enemies, after a short amount of time.

## Expected Behavior
The game should not freeze and you should be able to kill the stalfos to progress

## Steps to Reproduce
1. Create file 1
2. Use the debug menu to teleport to ganon's castle escape 3
3. Go through the room until the stalfos spawn
4. Wait for the game to freeze
(this happens in other areas with stalfos but this is the fastest way to recreate the bug)

**Comment by blawar** [comment]
I can confirm that this bug is fixed in the audio branch.  Will close issue once it is merged into master.
